src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -277,10 +277,31 @@ void SetParameter()
          Aux_Error( ERROR_INFO, "Incorrect parameter %s = %d !!\n", "CCSN_Is_PostBounce", CCSN_Is_PostBounce );
    }
 
-// CCSN_Min_AngRes must be larger than CCSN_Min_AngRes
-   if ( CCSN_Min_AngRes > 0.0  &&  CCSN_Max_AngRes > 0.0 )
-      if ( CCSN_Min_AngRes <= CCSN_Max_AngRes )
-         Aux_Error( ERROR_INFO, "%s = %d must be larger than %s = %d  !!\n", "CCSN_Min_AngRes", CCSN_Min_AngRes, "CCSN_Max_AngRes", CCSN_Max_AngRes );
+// check OPT__FLAG_REGION is enabled for CCSN_AngRes_Min/Max and convert degree to radian
+   const double DEG2RAD = M_PI/180.0;
+   if ( CCSN_AngRes_Min > 0.0 ) {
+      CCSN_AngRes_Min *= DEG2RAD;
```

Comment:
print information that `CCSN_AngRes_Min/Max` have been reset (`PRINT_WARNING()`).

Reply:
I added `PRINT_WARNING()` after unit conversion.